l10n_it_reverse_charge/models/account_move.py

```diff
@@ -127,8 +133,10 @@ def _rc_get_move_line_to_reconcile(self):
         else:
             raise UserError(_("Only inbound and outbound moves are supported"))
 
+        lines = self.line_ids
+        already_reconciled_lines = lines.filtered("reconciled")
```

Comment:
Come mai serve questo?

Reply:
Grazie della review!
In effetti non sembra servire per la migrazione quindi l'ho rimosso, puoi controllare se ora va meglio?